machine.js

```diff
@@ -129,15 +138,33 @@ chokidar.watch("projects.json").on("change", () => {
 
 /* Global max capacity */
 var maxCapacity = 1;
+var GPUsCapacity = [];
+var GPUCapacity = 1;
```

Comment:
Might be nicer to have it default to 0 (as the above code will set it properly if the machine has GPUs).